frontend/src/components/regions/UserActions/NewUserBurndownStats.jsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from "react";
```

Comment:
This file has a funny name. And it is in a funny place.

Who is this new user of which you speak?

And should this not be visible on one of the Teams tabs?

Reply:
It would be good to put this into a directory of it's own, it would do best with a PResentation.jsx file and an index.jsx file